test/ut/sdk/test_model_speedup.py

```diff
@@ -128,6 +128,18 @@ def generate_random_sparsity(model):
                              'sparsity': sparsity})
     return cfg_list
 
+def generate_random_sparsity_v2(model):
```

Comment:
maybe better to use ratio as a parameter, like `def generate_random_sparsity(model, layer_ratio):`, just personal opinion, the current implementation is fine to me.